mirrord/cli/src/execution.rs

```diff
@@ -189,6 +190,31 @@ impl MirrordExecution {
         })
     }
 
+    /// Some features don't make a lot of sense together with targetless agents, so if needed,
+    /// warn the user, if possible, disable unnecessary features.
```

Comment:
This new logic doesn't actually warn the user nor disable features, it just stops the execution with an error. The whole command is aborted, I think